demo/src/render_graph_generator/bloom_combine_pass.rs

```diff
@@ -77,6 +78,17 @@ pub(super) fn bloom_combine_pass(
             shaders::bloom_combine_frag::DescriptorSet0Args {
                 in_color: &sdr_image,
                 in_blur: &hdr_image,
+                config: &shaders::bloom_combine_frag::ConfigStd140 {
+                    tonemapper_type: match render_options.tonemapper_type {
+                        TonemapperType::None => 0,
```

Comment:
Can we make this a c-style enum so that we don't need to do a match?

Reply:
sure